cli.py

```diff
@@ -62,17 +59,17 @@ def add_arguments_for_verb(parser, verb: str) -> None:
     # Add argument groups for model specification (what base model to use)
     _add_model_specification_args(parser)
 
-    # Add argument groups for exported model path IO
-    _add_exported_input_path_args(parser, verb)
-    _add_export_output_path_args(parser, verb)
-    
     # Add argument groups for model configuration (compilation, quant, etc)
     _add_model_config_args(parser, verb)
 
     # Add thematic argument groups based on the subcommand
-    if verb in ["browser", "chat", "generate", "server"]:
+    if verb in GENERATION_VERBS:
+        _add_exported_input_path_args(parser)
         _add_generation_args(parser, verb)
+    if verb == "export":
+        _add_export_output_path_args(parser)
     if verb == "eval":
+        _add_exported_input_path_args(parser)
```

Comment:
Note that this arg input was previously dropped and is now added back in (Eval can take exported model inputs)